day3/flatten.py

```diff
@@ -0,0 +1,87 @@
+# Definition for a binary tree node.
+class TreeNode(object):
+    def __init__(self, x):
+        self.val = x
+        self.left = None
+        self.right = None
+
+## My Solution ##
+class Solution(object):
+    def flatten(self, root):
+        """
+        :type root: TreeNode
```

Comment:
Do you mean for the docstrings? I just copied from the leetcode question: https://leetcode.com/problems/flatten-binary-tree-to-linked-list/#/description